tokens/tokens.md

```diff
@@ -0,0 +1,144 @@
+# Aeternity Native Tokens
+
+## Overview
+
+We need to distinguish between a specification of a token, and a
+countable amount of the token.  In this document we use **ANT**
+(Aeternity Native Token) to represent the specification, and **native
+token** or simply **token** to represent the countable asset.
+
+Informally, when you need a new type of token for whatever purpose,
+you create an ANT describing the token, and then you can mint new
+tokens and start sending them to accounts.
+
+Owners of the tokens can spend them to other accounts, or trade them
+(atomically) for other assets with other accounts. If tokens are no
+longer wanted you can burn them.
+
+Only the owner of the ANT can mint new tokens, and is the owner
+decides that enough is enough, the ANT can be finalized, preventing
+more minting. An ANT can also be created with a final supply already
+minted and given to the owner of the ANT.
+
+An ANT can only be destroyed (TODO: design decision) if the total
+token supply of the ANT is zero (i.e., all tokens are burnt).
+
+Fees cannot be payed by native tokens, but only by aeons, but other
```

Comment:
--> "Fees cannot be paid by native tokens, but only by aeons. However, other
services can be paid for with native tokens. (TODO: Oracle query fees, etc?)" ...

Reply:
Somewhere it has to be made clear that "aeons" are not tokens. If one were able to trade tokens to aeons and back, one is most likely able to create new aeons in the system, which we don't want.

Reply:
Well, aeons are tokens, but not controlled by an ANT. You should surely be able to trade aeons for native tokens. There will not be new aeons created. Two parties simply agrees to make the trade, so the aeons come from one of the parties.